### PR TITLE
MGDOBR-1075: Review the metrics exposed by Fleet Manager

### PR DESCRIPTION
**Pull request by manstis**
[https://issues.redhat.com/browse/MGDOBR-1075](https://issues.redhat.com/browse/MGDOBR-1075)

This PR relates to the JIRA but is not its full implementation.

I just saw the error whilst chatting about metrics so thought it a good idea to fix now.

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [x] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [x] The layers in the `kustomize` folder have been updated accordingly.
- [x] All new functionality is tested
- [x] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [ ] Check if code and Github action workflows have been modified and if the modification is safe
- [ ] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.
- <b>Deploy BOT</b>
  Comment with `/deploy <dev,stable> when the PR has been merged to deploy to a target environment.

</details>

### CODE REVIEW

**Review by r00ta — APPROVED** [review]

File: manager/src/main/resources/application.properties [hunk 1]
```diff
@@ -2,9 +2,9 @@ event-bridge.k8s.orchestrator=${EVENT_BRIDGE_K8S_ORCHESTRATOR:openshift}
 event-bridge.manager.url=${EVENT_BRIDGE_MANAGER_URL:http://localhost:8080}
 
 # SLI Metric Names
-rhose.metrics-name.operation-total-count=managed_services_api_rhose_operation_count
-rhose.metrics-name.operation-success-total-count=managed_services_api_rhose_operation_success_count
-rhose.metrics-name.operation-failure-total-count=managed_services_api_rhose_operation_failure_count
+rhose.metrics-name.operation-total-count=managed_services_api_rhose_operation_total_count
```
Comment (robpblake) [inline manager/src/main/resources/application.properties:5]: @manstis Can I just double check that this doesn't result in the metric being exposed as `managed_services_api_rhose_operation_total_count_total` for example? I seem to remember that when I did this originally, the `_total` suffix was automatically appended by Micrometer framework, hence why it's missing from the metric name. 

For example if I check `dev` environment now, I see the following metrics exposed as an example:

```
managed_services_api_rhose_operation_success_count_total{operation="provision",resource="processor",} 146.0
managed_services_api_rhose_operation_success_count_total{operation="delete",resource="processor",} 146.0
managed_services_api_rhose_operation_success_count_total{operation="delete",resource="bridge",} 218.0
managed_services_api_rhose_operation_success_count_total{operation="provision",resource="bridge",} 220.0
managed_services_api_rhose_operation_success_count_total{operation="modify",resource="processor",} 30.0
managed_services_api_rhose_operation_count_total{operation="provision",resource="processor",} 146.0
managed_services_api_rhose_operation_count_total{operation="delete",resource="processor",} 146.0
managed_services_api_rhose_operation_count_total{operation="delete",resource="bridge",} 218.0
managed_services_api_rhose_operation_count_total{operation="modify",resource="bridge",} 13.0
managed_services_api_rhose_operation_count_total{operation="provision",resource="bridge",} 220.0
managed_services_api_rhose_operation_count_total{operation="modify",resource="processor",} 30.0
```

cheers,
Rob

Comment (manstis) [reply manager/src/main/resources/application.properties]: ... and now it looks like the code was already correct and the spreadsheet was out of date...

Closing this PR (as there are no changes!).

**Comment by manstis** [comment]
The "RHOSE SLO/SLI - Working Document" metrics spreadsheet contained the wrong names.

RHOSE codebase had correct names (based on how Micrometer appends `_total` suffixes). See PR [discussion](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/1183/commits/41bae03ad8e1997d202ff8494a4d71dc5eb0955e#r984384337).

File: manager/src/main/resources/application.properties (hunk 1 above)
Comment (robpblake) [reply manager/src/main/resources/application.properties]: Thanks @manstis 👍

**Pull request closed without merging**